### PR TITLE
[ENH] Add page on contributing PRs (with new prefixes)

### PR DESCRIPTION
**Pull request by alyssadai**
Closes https://github.com/neurobagel/project/issues/139
See also https://github.com/neurobagel/.github/pull/5

Details of PR prefix meanings added, as well as basic info on PR review emojis.

Documentation-specific, simplified version of PR template without the template preamble (since generally developers update the docs) also added, with suggested prefixes limited to those relevant to documentation PRs.

## Checklist

- [x] PR has an interpretable title with a prefix (`[ENH]`, `[BUG]`, `[DOC]`, `[INFRA]`, `[MAINT]`) _(see https://neurobagel.org/contributing/pull_requests for more info)_
- [x] PR links to Github issue with mention `Closes #XXXX`
- [ ] Tests pass
- [ ] Code is properly formatted

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+-
+-
+
+<!-- To be checked off by reviewers -->
+## Checklist
+
+- [ ] PR has an interpretable title with a prefix (`[ENH]`, `[FIX]`, `[REF]`, `[TST]`, `[CI]`, `[MNT]`, `[INF]`) _(see https://neurobagel.org/contributing/pull_requests for more info)_
+- [ ] PR links to GitHub issue with mention `Closes #XXXX`
+- [ ] Checks pass
```
Comment (surchs) [inline .github/pull_request_template.md:24]: can we also have the "Code formatted" bit from the other template https://github.com/neurobagel/.github/pull/5

Comment (alyssadai) [reply .github/pull_request_template.md:24]: Leaving this out as it would be part of the checks! :)

File: mkdocs.yml [hunk 2]
```diff
@@ -42,6 +42,8 @@ nav:
       - Pipeline specific instructions:
         - fmriprep: "nipoppy/workflow/proc_pipe/fmriprep.md"
         - mriqc: "nipoppy/workflow/proc_pipe/mriqc.md"
+  - Contributing:
+    - Pull requests: "contributing/pull_requests.md"
```
Comment (surchs) [inline mkdocs.yml:46]: Is there something we can combine with this page so the navbar on the left doesn't look so lonely with just the `Pull requests` section?

File: docs/contributing/pull_requests.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+- `[ENH]:` Feature improvements or additions
+- `[REF]:` Refactoring existing code
+- `[TST]`: Updating or adding a test
+- `[CI]`: Automation-related changes
+- `[MNT]`: General maintenance not covered by `[REF]`, `[TST]`, or `[CI]`
+- `[INF]`: Software or graph infrastructure-related changes
+- `[FIX]`: Bug fixes
+- `[MODEL]`: Updates or changes related to the Neurobagel data model
+- `[DOC]`: Documentation-only changes to a code repo (READMEs, within-code documentation, etc.) 
+_Note: do not use for changes to the [`neurobagel/documentation`](https://github.com/neurobagel/documentation) repo_
```
Comment (surchs) [inline docs/contributing/pull_requests.md:14]:
```suggestion
_Note: do not use for changes to the [`neurobagel/documentation`](https://github.com/neurobagel/documentation) repo_. See below for PR prefixes for the documentation repo.
```

**Review by surchs — APPROVED** [review]
Looks good @alyssadai. Just a :cherries: comment on the navbar on the left. Maybe we can combine something here - otherwise this navbar is looking a bit sparse.

File: mkdocs.yml (hunk 2 above)
Comment (alyssadai) [reply mkdocs.yml:46]: I see what you mean. I’m not entirely sure what existing section would also fall under “Contributing” TBH - do you have something in mind? I saw this tab also as a placeholder that we can add to with other process-related things in the future.

One option would be to put the separate page we originally had describing “Our team” (https://github.com/neurobagel/documentation/blob/main/docs/team.md - not currently visible in the nav) under this tab, so we can also shorten the home page a bit. Does that sound good?

Comment (alyssadai) [reply mkdocs.yml:46]: This has been implemented!

**Pull request merged**